packages/terra-date-picker/tests/wdio/date-picker-spec.js

```diff
@@ -326,7 +326,7 @@ Terra.describeViewports('Date Picker', ['medium'], () => {
 
     describe('Invalid DatePickerField', () => {
       before(() => {
-        browser.url('/#/raw/tests/terra-date-picker/date-picker/date-picker-field');
+        browser.refresh();
         browser.click('#validity-toggle');
       });
 
```

Comment:
We should avoid using refresh when possible. What are these changes fixing with the theme repo?

Reply:
Looking through the commit history, if we can re-click the `#validity-toggle` button to adjust the state, it would be cheaper from a testing perspective. Using `refresh` kills the current session, sends a new session request, (when using the selenium grid, this can cause tests to be pending till a node becomes available) and then refreshes the page to reset the state.

Reply:
In the current way the fusion theme repo run the tests, the pages are keeping the toggle states between screenshots so the later tests (incomplete and incomplete invalid) are displaying wrong.

I've only tested the fusion theme repo with this so far.

Reply:
Fixed here https://github.com/cerner/terra-framework/pull/962/commits/78ec9821e8de85d34678c4514e45c0b21db37209